pkg/csi/types/constants.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Since this file is just renamed/moved, the copyright year should stay at 2018.

Reply:
done. fixed.